superduperdb/base/build.py

```diff
@@ -87,14 +94,16 @@ def build(uri, mapping):
             if cfg.artifact_store is not None
             else databackend.build_artifact_store()
         ),
-        distributed_client=dask_client(
-            cfg.cluster.dask_scheduler,
-            local=cfg.cluster.local,
-            serializers=cfg.cluster.serializers,
-            deserializers=cfg.cluster.deserializers,
-        )
-        if cfg.mode == 'production'
-        else None,
+        compute=(
```

Comment:
can you create a factory_method which will create a compute instances 

may be we can do this later, im fine